app/views/taxons/_deletion_warning.html.erb

```diff
@@ -0,0 +1,41 @@
+<% if tagged.any? || children.any? %>
+<div class="alert alert-warning clearfix" role="warning">
+  <p>
+  This taxon has content tagged to it, deleting it will cause the content to not
```

Comment:
I was thinking in having some approval on the wording first before moving it to a `I18n`, have to talk with someone to see what would be the correct wording here.